cmd/controller/main.go

```diff
@@ -25,47 +25,64 @@ import (
 var version = "dev"
 var name = "deviceplane-controller"
 
+//commit for test
 var (
 	addr = kingpin.
 		Flag("addr", "").
 		Default(":8080").
+		OverrideDefaultFromEnvar("ADDR").
 		String()
 	mysql = kingpin.
 		Flag("mysql", "").
 		Default("deviceplane:deviceplane@tcp(localhost:3306)/deviceplane?parseTime=true").
+		OverrideDefaultFromEnvar("MYSQL_CONNECTION_STRING").
 		String()
+	enableSignUp = kingpin.
+			Flag("enableSignUp", "").
+			OverrideDefaultFromEnvar("ENABLE_SING_UP").
+			Bool()
 	statsdAddress = kingpin.
 			Flag("statsd", "").
 			Default("127.0.0.1:8125").
+			OverrideDefaultFromEnvar("STATS_ADDRESS").
 			String()
 	allowedOrigins = kingpin.
 			Flag("allowed-origin", "").
+			OverrideDefaultFromEnvar("ALLOWED_ORIGINS").
 			Strings()
 	emailProvider = kingpin.
 			Flag("email-provider", "").
 			Default("none").
+			OverrideDefaultFromEnvar("EMAIL_PROVIDER").
 			String()
 	emailFromName = kingpin.
 			Flag("email-from-name", "").
 			Default("Deviceplane").
+			OverrideDefaultFromEnvar("EMAIL_FROM_NAME").
 			String()
 	emailFromAddress = kingpin.
 				Flag("email-from-address", "").
+				OverrideDefaultFromEnvar("EMAIL_FROM_ADDRESS").
 				String()
 	allowedEmailDomains = kingpin.
 				Flag("allowed-email-domain", "").
+				OverrideDefaultFromEnvar("ALLOWED_EMAIL_DOMAINS").
```

Comment:
How would this work with multiple email domains?  We currently enable 2 different email domains via multiple flags, but not sure how that'd work via env vars.